### PR TITLE
[DF] Better error message for Vary+Fill+no Reset method

### PR DESCRIPTION
**Pull request by eguiraud**

### CODE REVIEW

File: tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx [hunk 1]
```diff
@@ -516,6 +516,17 @@ public:
       UnsetDirectoryIfPossible(result.get());
       return FillHelper(result, fObjects.size());
    }
+
+   // This overload is selected if HIST does not have a Result method, i.e. we cannot
```
Comment (ikabadzhov) [inline tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx:520]: `Reset`, not `Result`, right?

**Comment by ikabadzhov** [comment]
Everything else looks good to me.

File: tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx (hunk 1 above)
Comment (eguiraud) [reply tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx:520]: definitely, thanks!

**Review by ikabadzhov — APPROVED** [review]
lgtm!

**Pull request merged**